data/tutorial/contents.md

```diff
@@ -220,7 +220,7 @@ A convenience function for adding a timestamp to a `C.t` value:
 The merge operation for `Lww_register` is slightly different than the ones covered so far. It will not attempt to merge any values, instead it will pick the newest value based on the attached timestamp.
 
 ```ocaml
-    let merge ~old (a, timestamp_a) (b, timestamp_b) =
+    let merge ~old:_old (a, timestamp_a) (b, timestamp_b) =
```

Comment:
```suggestion
    let merge ~old:_ (a, timestamp_a) (b, timestamp_b) =
```
Just a suggestion, but I'd normally avoid naming an ignored param that is already named like this.